consensus/merger.go

```diff
@@ -73,6 +73,7 @@ func (m *Merger) ReachTTD() {
 	}
 	rawdb.WriteTransitionStatus(m.db, blob)
 	log.Info("Left PoW stage")
+	log.Info("WE JUST MERGED LFGGGGGGGGGGGGG")
```

Comment:
Meh. This isn't classy, IMO. 
I'd rather go for something more subtle. Some unicode emoticon, or maybe some quote, or maybe some fine ascii art.

Reply:
🐼🐼🐼

Reply:
"A small step for a man, a huge leap for mankind"
"The internet of money should not cost $0.05 per transaction - @vbuterin"
"Ethereum ESG ETF incoming"
"I don't believe a second, compatible implementation of Bitcoin will ever be a good idea - Satoshi Nakomoto"